lib/wallaby/phantom.ex

```diff
@@ -1,13 +1,45 @@
 defmodule Wallaby.Phantom do
+  @moduledoc """
+  Wallaby driver for PhantomJS.
+
+  ## Usage
+
+  Start a Wallaby Session using this driver with the following command:
+
+  ```
+  {:ok, session} = Wallaby.start_session(driver: Wallaby.Phantom)
+  ```
+
+  ## Notes
+
+  This driver requires PhantomJS be installed in your path. You can install PhantomJS through NPM or your package manager of choice:
```

Comment:
unless you specify the phantomjs path (which is what I usually do, all to the same npm package so everyone runs the same version) - maybe we could make that clearer here?